### PR TITLE
Remove hardcoded family

### PR DESCRIPTION
**Pull request by fitn**
| Q                 | A
| ----------------- | ---
| Added Specs       | N
| Added Behats      | N
| Travis CI is ok   | running
| Changelog updated | Y

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Controller/FamilyController.php [hunk 1]
```diff
@@ -171,17 +177,17 @@ public function createAction()
...
-     *
-     * @param Family $family
+     * @param int $familyId
      *
      * @Template
      * @AclAncestor("pim_enrich_family_index")
      *
      * @return array
      */
-    public function editAction(Family $family)
```
Comment (solivier) [inline src/Pim/Bundle/EnrichBundle/Controller/FamilyController.php]: it does not work with the entity ?

Comment (jjanvier) [inline src/Pim/Bundle/EnrichBundle/Controller/FamilyController.php]: no, not if the guy implements its own Family

**Comment by jjanvier** [comment]
GTM

**Comment by jjanvier** [comment]
I launch the CI

**Comment by nidup** [comment]
GTM if CI ok

**Comment by solivier** [comment]
GTM

**Pull request merged**